usr/sbin/truenas-install

```diff
@@ -79,8 +78,8 @@ check_is_swap_safe()
 	for _disk ; do
 	  sector_size=$(blkid -i /dev/${_disk} | sed "s/.*LOGICAL_SECTOR_SIZE=\"\([0-9]\+\)\"/\1/")
 	  sectors=$(cat /sys/block/${_disk}/size)
-	    if [ $((sector_size * sectors)) -lt ${MIN_SWAPSAFE_MEDIASIZE} ] &&
-		grep -qF "0" /sys/block/${_disk}/removable ; then
+	    if [ $((sector_size * sectors)) -lt ${MIN_SWAPSAFE_MEDIASIZE} ] ||
+		grep -qF "1" /sys/block/${_disk}/removable ; then
 		SWAP_IS_SAFE="NO"
```

Comment:
prolly a st00pid question, but I assume that `SWAP_IS_SAFE` defaults to `YES` when we get to `check_is_swap_safe()` ?

Reply:
It defaults to being unset (unless it was already set in the environment, either by a previous call or explicitly by an advanced user). Assuming it was unset, this initial loop sets it to NO if we find one of the selected disks does not meet the requirements. If it's still unset after the loop, then we set it to YES.

Looks like there's one more comment I need to update below.